keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -558,23 +557,29 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-  TBD
+  Enabling this feature gate moves the responsibility for handling mixed protocols from the Kubernetes API server to the specific CPI. See the provider-specific details at `API change and upgrade/downgrade situations`.
 
 * **What specific metrics should inform a rollback?**
 
-  TBD
+  As all providers either already have adjusted their error messages or intend to, enabling this feature gate may lead to a CPI error. In that case, remediation would be to disable the gate.
```

Comment:
I suppose also if LB traffic didn't route correctly, that would be a good reason to roll back?

Reply:
yeah, in this case the LB traffic ONLY works correctly for one protocol and not for the other